example/src/main/scala/example/HelloWorldAdvanced.scala

```diff
@@ -21,10 +21,13 @@ object HelloWorldAdvanced extends App {
     case Method.GET -> !! / "utc"    => clock.currentDateTime.map(s => Response.text(s.toString))
   }
 
+  private val errorHandler: Throwable => ZIO[Any, Nothing, Unit] = cause => UIO(println(cause))
```

Comment:
We should never use `UIO(println(cause)`
Also we should not update example for minor changes

Reply:
ok, done